tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+export test_key="aatest"
+export SUPPORTED_TEE_HYPERVISORS=("qemu-sev" "qemu-snp" "qemu-tdx" "qemu-se")
+export SUPPORTED_NON_TEE_HYPERVISORS=("qemu")
+
+setup() {
+	if ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_TEE_HYPERVISORS[@]} " ]] && ! [[ " ${KATA_HYPERVISOR} " =~ " ${SUPPORTED_NON_TEE_HYPERVISORS} " ]]; then
```

Comment:
am I correct that this will also run in Non-Tees?

Reply:
and for non-tees should we set image annotations?

Reply:
Hi @GabyCT ! Yes, it should run in non-TEE jobs too. Actually, it will be first enabled for k8s on AKS, so in a non-TEE job.

Yes for the 2nd question tool. The non-TEE jobs kata's configuration.toml set the vanilla image that doesn't have the attestation components. It will use the rootfs-image-confidential image. I'm almost sure hypervisor (qemu) and kernel will be used the vanilla ones, but let's check.

Reply:
Yes non-tee test will use the confidential image but the standard kernel and hypervisor